include/dali/core/tuple_helpers.h

```diff
@@ -233,6 +233,34 @@ using tuple_generator_t = typename tuple_generator_type<type_generator, Sequence
 using detail::apply;
 using detail::apply_all;
 
+
+namespace detail {
+
+template<typename T, typename Tuple>
+struct contains;
```

Comment:
Tbh. I liked the old name `has_type` better but it is just a matter of preference, I guess. I know this name was suggested by other reviewer so no point in changing it back and forth :)